src/routes.js

```diff
@@ -1,9 +1,44 @@
-const { Router } = require("express");
+const { Router, application } = require("express");
```

Comment:
Não vi esse `application` ser utilizado. Nunca deixa variáveis ou recursos não utilizados no código.

```suggestion
const { Router } = require("express");
```